tools/asmrefine/SimplExport.thy

```diff
@@ -1105,15 +1110,18 @@ fun emit_func_body ctxt outfile eparams name = let
     else (emit outfile ("1 Basic Ret 0");
           emit_body ctxt outfile eparams body 2 "1" ("ErrExc", "ErrExc")
             |> snd |> prefix "EntryPoint " |> emit outfile
+(*
           handle TERM (s, ts) => raise TERM ("emit_func_body: " ^ name ^ ": " ^ s, body :: @{term True} :: ts)
             | TYPE (s, Ts, ts) => raise TYPE ("emit_func_body: " ^ name ^ ": " ^ s, Ts, body :: @{term True} :: ts)
             | Empty => raise TERM ("emit_func_body: Empty", [body]))
+*)
+)
```

Comment:
Did you mean to leave this commented out?

Reply:
Embarrassing medical incident in the git surgery (and the tracing too), thank you for spotting! Will fix.

Reply:
Though I'll wait until regression proof tests complete, as these don't change the success criteria.